Remove reference to 'trunk'

milljm opened the pull request:
git grep trunk now returns zero hits. huray!

Closes #25862

Commit:
Remove reference to 'trunk'

git grep trunk now returns zero hits. huray!

Closes #25862

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1845678/) on d59a134 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/25867/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1845696/) on d59a134 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>d2dcda</th>
      <th colspan=4>#25867 d59a13</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>85.63%</td>
      <td>85.64%</td>
      <td>+0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>92364</td>
      <td>92369</td>
      <td>+5</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15498</td>
      <td>15493</td>
      <td>-5</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/25867/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/25867/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/25867/coverage/xfem)

</details>



This comment will be updated on new commits.

Commit:
Adding libxcb-* to WSL instructions

Both a user and myself have needed to install this package to make
Peacock work. So add it.

Closes #25862
